docs/user/how-to/write-plans.rst

```diff
@@ -0,0 +1,203 @@
+Writing bluesky plans for blueapi
+=================================
+
+**Please read the following completely and carefully, as you risk losing data if plans are written incorrectly.**.
+
+For an introduction to bluesky plans and general forms/advice, `see the
+bluesky documentation <https://nsls-ii.github.io/bluesky/plans.html>`__.
+Blueapi has some additional requirements, which are explained below.
+
+Generally, if a ``MsgGenerator`` includes at least one
+``open_run`` and ``close_run``, it is a ``plan``, a complete description
+of an experiment. If it does not, it is a ``stub``.
+
+Plans
+~~~~~
+
+A ``plan`` is a complete data collection proceedure.
```

Comment:
Rather than a whole section, could just link to here: https://blueskyproject.io/bluesky/plans.html

Reply:
I see, that's confusing then. Maybe we need a different word? Or just define a stub here too?